src/app/about/about.component.ts

```diff
@@ -10,5 +10,6 @@ import { Component } from '@angular/core';
     templateUrl: './about.component.html'
 })
 export class AboutComponent {
+    open: Boolean = false;
```

Comment:
oh you're right, it would fail if not declared in AoT. Thanks again for this PR.